Update doc chapter "Getting Started" (#527)
adapter-hub/adapter-transformers: adapter_docs/training.md

## Train a Task Adapter

> Training a task adapter module on a dataset only requires minor modifications from training a whole model.

Reviewer: ```suggestion
Training a task adapter module on a dataset only requires minor modifications compared to training the entire model.
```
For consistency, I would use "entire model" instead of "whole model" as you have used "entire model" everywhere else.

## AdapterTrainer

Similar to the `Trainer` class provided by HuggingFace, `adapter-transformers` provides an `AdapterTrainer` class. This class is only
intended for training adapters. The `Trainer` class should still be used to fully fine-tune models. To train adapters with the `AdapterTrainer`

> class, simply initialize it the same way you would initialize the `Trainer` class,e.g.: 

Reviewer 2: ```suggestion
class, simply initialize it the same way you would initialize the `Trainer` class, e.g.: 
```
